docs/Concepts/Multitenancy/Overview.md

```diff
@@ -63,46 +53,66 @@ authServer -- tb
+    --multitenancy \
+    --plugins file:///<path>/<to>/plugins.json
+```
+
+In the command, `plugins.json` is the [plugin settings file](../../HowTo/Configure/Plugins.md) that
+contains the [JSON RPC Security plugin definition](../../HowTo/Configure/Plugins.md#plugindefinition).
+View the [security plugins documentation] for more information about how to configure the JSON RPC
+Security plugin.
 
 ## Access Token Scope
 
-JSON RPC Security plugin enables `geth` JSON RPC API server to be an OAuth2-compliant resource server. 
-A client must first obtain a pre-authenticated access token from an authorization server then 
-presents the access token (via `Authorization` HTTP request header) when calling an API.
+The JSON RPC Security plugin enables the `geth` JSON RPC API server to be an OAuth2-compliant
+resource server. A client must first obtain a pre-authenticated access token from an authorization
+server, then present the access token (using an `Authorization` HTTP request header) when calling an
+API.
 
-The value of the scope encoded in an access token (in case of JWT) or in an introspection response
-(in case of OAuth2 Token Introspection API) contains [RPC scope](../../Reference/Plugins/security/For-Users/#oauth2-scopes)
+The value of the scope encoded in an access token (in case of JWT), or introspection response
+(in the case of the OAuth2 Token Introspection API) contains the [RPC scope](../../Reference/Plugins/security/For-Users.md#oauth2-scopes)
 and tenant scope which has the following URL-based syntax:
 
-```
+```text
     "private://0x0/_/contracts?owned.eoa=0x0&from.tm=[tm-pubkey]"
 ```
 
-in which:
+In the syntax, `tm-pubkey` is the URL-encoded value of the Privacy Manager public key.
 
-* `tm-pubkey`: URL-encoded value of the Privacy Manager public key
-
-For example, a client owns two Privacy Manager public keys `PUBKEY1` and `PUBKEY2`, 
+For example, for a client that owns two Privacy Manager public keys `PUBKEY1` and `PUBKEY2`,
 an authorization server operator would setup and grant the following scopes to the client:
 
-```
+```text
     private://0x0/_/contracts?owned.eoa=0x0&from.tm=PUBKEY1
     private://0x0/_/contracts?owned.eoa=0x0&from.tm=PUBKEY2
 ```
 
-A client presenting an access token containing the above scopes has the full access (read/write/create)
-to private contracts in which `PUBKEY1` and/or `PUBKEY2` are the participants.
\ No newline at end of file
+A client presenting an access token containing the above scopes has full access (read/write/create)
+to private contracts in which `PUBKEY1` and `PUBKEY2` are participants.
+
+<!--links-->
+[Multi-tenancy]: ../../HowTo/Use/Multitenancy.md
+[scope]: #access-token-scope
+[scoped access tokens]: #access-token-scope
+[pre-authenticated access tokens with the authorized scope]: #access-token-scope
+[security plugins documentation]: ../../Reference/Plugins/security/For-Users.md#configuration
+[Tessera]: https://docs.tessera.consensys.net/en/stable/
```

Comment:
Prefer the link to the root without language and version, there's a redirect to handle that automatically.
```suggestion
[Tessera]: https://docs.tessera.consensys.net/
```